dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -225,27 +233,57 @@ def SignalHandler(*unused_argvs):
   sys.exit(0)
 
 
+def SetupLogging():
+  """Sets up a logging handler with dftimewolf's custom formatter."""
+  # Clear root handlers (Turbinia is setting them)
```

Comment:
maybe you can just say "dependent libraries are configuring the root logger', since turbina may change it's way and other libraries may be messing with it too....

Reply:
True. Done.